README.md

```diff
@@ -28,49 +23,26 @@ mvn archetype:generate \
+- **`projectTitle`**: Descriptive project name (e.g. `My App`)
+- **`projectName`**: Technical project name, used for building AEM paths (like `/content/${projectName}/en`, e.g. `myapp`)
+- **`groupId`**: ID which uniquely identifies your group and project, should start with a reversed domain name you control (e.g. `com.mycompany`)
+- **`optionFrontend`**: Frontend framework to use in the generated project (either `angular` or `react`)
 
 See [`archetype-metadata.xml`](./src/main/resources/META-INF/maven/archetype-metadata.xml) for the full list possible parameters.
 
-## Modules
+## Documentation
 
-Modules of the generated project is defined in [src/main/resources/archetype-resources](src/main/resources):
-
-* [core](core/): OSGi bundle containing:
-  * Java classes (e.g. Sling Models, Servlets, business logic)
-* [ui.apps/src/main/content/jcr_root/apps](content/jcr_root/apps/):
-  * AEM components with their scripts and dialog definitions
-* [ui.content/src/main/content/jcr_root/conf](content/jcr_root/conf/):
-  * AEM content package with editable templates stored at `/conf`
-* [ui.content/src/main/content/jcr_root/content](content/jcr_root/content/):
-  * AEM content package containing sample content (for development and test purposes)
-* [angular-app](angular-app/): Angular application in case frontend chosen is set to be "angular" at project generation
-* [react-app](react-app/): React application in case frontend chosen is set to be "react" at project generation
-* [all](all/): Combines all modules to be installed as content package in AEM
-
-## Provided Maven profiles
-The generated maven project support different deployment profiles when running the Maven install goal `mvn install` within the reactor.
-
-Id                        | Description
---------------------------|------------------------------
-autoInstallBundle         | Install core bundle with the maven-sling-plugin to the felix console
-autoInstallPackage        | Install the ui.content and ui.apps content package with the content-package-maven-plugin to the package manager to default author instance on localhost, port 4502. Hostname and port can be changed with the aem.host and aem.port user defined properties.
-autoInstallPackagePublish | Install the ui.content and ui.apps content package with the content-package-maven-plugin to the package manager to default publish instance on localhost, port 4503. Hostname and port can be changed with the aem.host and aem.port user defined properties.
-
-The profile `integrationTests` is also available for the verify goal, to run the provided integration tests on the AEM instance.
+- [How to use the generated AEM project](./src/main/resources/archetype-resources/README.md)
+- [How to use the generated Angular app](./src/main/resources/archetype-resources/angular-app/README.md)
+- [How to use the generated React app](./src/main/resources/archetype-resources/react-app/README.md)
 
 ## Development
 
 ### Building
```

Comment:
[Minor Suggestion] - We can put this part in Development.md -
https://github.com/samuelmeuli/aem-spa-project-archetype/tree/readmes#development
(Building and Releasing)

Reason-
```For npm modules the content of README.md is put on the main npm page of the module, and it is a rare occasion, that a user would be interested how to build or run-tests of such a a library and would be more interested in usage```

But, I will leave it for you to decide.

Reply:
Now that I think about it, 3 files - Readme, Contributing and Development might be too much.
So, we could leave it like that :D

Reply:
:+1::blush: